### PR TITLE
Modal View Cancelation/Deletion of Mentor's event registration

### PR DESCRIPTION
**Pull request by 1620prabn**
# modal view for cancelation/deletion of mentor's event registration 

Please provide a short summary explaining what this PR is about.

Fixes # (#141 )

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/ModalWithCancelRegistration/ModalWithCancelRegistration.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import './ModalWithCancelRegistration.css';
+import Modal from '../Modal/Modal';
+
+function ModalWithCancelRegistration(props) {
+  return (
+    <div className="cancel-reg-modal">
+      <Modal title="Cancel Registration">
+        <div className="cancel-reg-textfield">{props.text}</div>
+        <div className="cancel-reg-submit-btn">
+          <Button>Cancel</Button>
```
Comment (tpetrina) [inline src/client/components/ModalWithCancelRegistration/ModalWithCancelRegistration.js:13]: Propagate `onClick` through props as well.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Good start, just a little bit more.

File: src/client/components/ModalWithCancelRegistration/ModalWithCancelRegistration.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.cancel-reg-modal .modal-wrapper {
```
Comment (tpetrina) [inline src/client/components/ModalWithCancelRegistration/ModalWithCancelRegistration.css:1]: Isn't this too much CSS when the base Modal is good enough?

**Review by tpetrina — APPROVED** [review]
Nice.

**Review by MaherHussain — APPROVED** [review]

**Pull request merged**